deployments/epp/staging/deployment-scaledobject.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: epp-import-worker-epp
+spec:
+  scaleTargetRef:
+    name: epp-import-worker-epp
+    namespace: epp--staging
+  minReplicaCount: 1
+  maxReplicaCount: 10
+  advanced:
+    horizontalPodAutoscalerConfig:
+      behavior:
+        scaleUp:
+          stabilizationWindowSeconds: 30
+          policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 60
+        scaleDown:
+          stabilizationWindowSeconds: 60
+          policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 60
+  triggers:
+    - type: temporal
+      metadata:
+        namespace: epp--staging
+        taskQueue: epp
+        targetQueueSize: "1"
```

Comment:
Each EPP worker is able to take multiple tasks concurrently. You might need to rework these exact values to avoid creating lots of pods, causing large cluster scale up (and $ spend), see [these values](https://github.com/elifesciences/elife-flux-cluster/blob/7783252454b98b55546b19ce5b44d8eb6048bcd7/kustomizations/apps/epp/import/worker-epp/deployment-worker-epp.yaml#L40-L43) for the defaults